### DISCUSSION TITLE
[ENDPOINT]: Proposed design change for /shops/{shop_id}/listings/active

### DISCUSSION THREAD

**Discussion by priyam1103** [etsy/open-api#703] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing tickets

### Current Endpoint Design

The listings that are returned by this API do not have `review_average` and `review_count` included.

### Proposed Endpoint Design Change

Include `review_average` and `review_count` in the API result.

`https://openapi.etsy.com/v3/application/shops/{shop_id}/listings/active`
`https://openapi.etsy.com/v3/application/listings/batch`

There are several other endpoints where we think this should be added.

```
https://openapi.etsy.com/v3/application/listings/active
https://openapi.etsy.com/v3/application/shops/{shop_id}/listings/featured
https://openapi.etsy.com/v3/application/listings/{listing_id}
```

### Why are you proposing this change?

To get the review we will have to make an extra API call for each listing to get the reviews from the `https://openapi.etsy.com/v3/application/listings/{listing_id}/reviews` API, which is not at all efficient.

Also `https://openapi.etsy.com/v3/application/listings/{listing_id}/reviews` does not require OAuth to get the review so we think review details can be added above mention endpoints and shops endpoints also have `review_average` and `review_count`

**Comment by dusariudaykumar** [comment]
+1
this would be most useful for us too @etsyachristensen

**Comment by codyjmcguffie** [comment]
+1 Yes please! This will help us greatly also!

**Comment by priyam1103** [comment]
@etsyachristensen Do we have any updates on this, please?

**Comment by benjymouse** [comment]
+1 would be very good

**Comment by chrismeats** [comment]
+1 @etsyachristensen Any chance of this being added?
Getting these numbers from reviews endpoint will burn through the daily limit pretty quick.

**Reply by Gareth064** [reply]
@etsyachristensen doesn't work for Etsy anymore.

Etsy Devs don't monitor this GitHub much anymore. If you want to try to get an update on something like this then it's better to email developer@etsy.com to see if they can tell you.

**Comment by kiernangs** [comment]
> @etsyachristensen doesn't work for Etsy anymore.
> 
> Etsy Devs don't monitor this GitHub much anymore. If you want to try to get an update on something like this then it's better to email [developer@etsy.com](mailto:developer@etsy.com) to see if they can tell you.



> Etsy Devs don't monitor this GitHub much anymore.

Ugh really? one more place that etsy can ignore input.  Why!?

**Comment by emergingdzns** [comment]
We really need this too! I've requested via email I think months ago but no response.
